tests/e2e/trustyai_test.go

```diff
@@ -18,12 +25,52 @@ func trustyAITestSuite(t *testing.T) {
 		ComponentTestCtx: ct,
 	}
 
+	// TrustyAI requires some CRDs that are shipped by Kserve
+	t.Run("Enable Kserve", componentCtx.enableKserve)
+
 	t.Run("Validate component enabled", componentCtx.ValidateComponentEnabled)
 	t.Run("Validate operands have OwnerReferences", componentCtx.ValidateOperandsOwnerReferences)
 	t.Run("Validate update operand resources", componentCtx.ValidateUpdateDeploymentsResources)
 	t.Run("Validate component disabled", componentCtx.ValidateComponentDisabled)
+
+	t.Run("Disable Kserve", componentCtx.disableKserve)
 }
 
 type TrustyAITestCtx struct {
 	*ComponentTestCtx
 }
+
+func (c *TrustyAITestCtx) enableKserve(t *testing.T) {
+	g := c.NewWithT(t)
+
+	g.Update(
+		gvk.DataScienceCluster,
+		c.DSCName,
+		testf.Transform(`.spec.components.%s.managementState = "%s"`, componentApi.KserveComponentName, operatorv1.Managed),
+	).Eventually().Should(
+		jq.Match(`.spec.components.%s.managementState == "%s"`, componentApi.KserveComponentName, operatorv1.Managed),
+	)
+
+	g.List(gvk.Kserve).Eventually().Should(And(
+		HaveLen(1),
+		HaveEach(And(
+			jq.Match(`.status.conditions[] | select(.type == "Ready") | .status == "%s"`, metav1.ConditionTrue),
+		)),
+	))
+}
+
+func (c *TrustyAITestCtx) disableKserve(t *testing.T) {
+	g := c.NewWithT(t)
+
+	g.Update(
+		gvk.DataScienceCluster,
+		c.DSCName,
+		testf.Transform(`.spec.components.%s.managementState = "%s"`, componentApi.KserveComponentName, operatorv1.Managed),
```

Comment:
Should this be `operatorv1.Removed`? to disable the component

Reply:
fixed